crud.py

```diff
@@ -0,0 +1,41 @@
+import models
+import schemas
+from sqlalchemy.orm import Session
+
+
+def get_all_authors(db: Session):

```

Comment:
The `get_all_authors` function should accept `skip` and `limit` parameters to support pagination, as indicated by the usage in `main.py`. Consider adding these parameters and using them in the query.